packages/devtools/src/FlattenView.tsx

```diff
@@ -19,6 +20,11 @@ const useStyles = makeStyles({
     columnGap: '4px',
     fontFamily: 'system-ui',
     fontSize: '12px',
+
+    position: 'sticky',
+    top: 0,
+    zIndex: 99,
```

Comment:
```suggestion
    zIndex: 1,
```

is 99 really required?